code/_onclick/hud/hud.dm

```diff
@@ -218,6 +218,8 @@ var/global/obj/screen/clicker/catcher = new()
 		shade_hud()
 	else if(isslime(mymob))
 		slime_hud()
+	else if(istype(mymob, /mob/living/simple_animal/slime))
```

Comment:
I've pondered whether isslime should include them, but I thought it might break a few checks if it did.